Build.fsx

```diff
@@ -126,9 +125,9 @@ Target "TestOnly" (fun _ ->
                                                 ResultSpecs = ["NUnit3TestResult.xml;format=nunit2"] })
 )
 
-Target "BuildAndTestOnly" (fun _ -> ())
-Target "Build" (fun _ -> ())
-Target "Test"  (fun _ -> ())
+Target "BuildAndTestOnly" DoNothing
+Target "Build" DoNothing
+Target "Test"  DoNothing
```

Comment:
Currently they are needed. Let's discuss that below and if we agree to strip `BuildOnly` and `TestOnly`, I'll refactor them.